packages/astro/test/units/routing/manifest.test.js

```diff
@@ -144,8 +144,7 @@ describe('routing - createRouteManifest', () => {
 
 		assertRouteRelations(getManifestRoutes(manifest), [
 			['/', '/[...rest]'],
-			['/static', '/static-'],
-			['/static-', '/[dynamic]'],
```

Comment:
This check shouldn't be removed, it should no longer be truncated but still be there

Reply:
It is combined into a single rule, isn't it?

Reply:
No, this is checking two different rules
- `/static` comes before `/static-[dynamic]` (fully static vs mixed static + dynamic)
- `/static-[dynamic]` comes before `/[dynamic]` (fully dynamic vs mixed static + dynamic)